actions/ctf/build-test-image/README.md

```diff
@@ -0,0 +1,4 @@
+# ctf/build-test-image
```

Comment:
i don't know if it will like the `ctf/` part it seems like a different path - can you do `ctf-`?

Reply:
@erikburt do you still any potential issue with this naming convention?

Reply:
I don't foresee any issues.

Reply:
fixed in https://github.com/smartcontractkit/.github/pull/600/commits/1e334563bf942b5647d4f3240596259beb877218